examples/fixture/asm/r1cs/poseidon.asm

```diff
@@ -0,0 +1,185 @@
+@ noname.0.7.0
```

Comment:
this file is wrong, not sure why it's displaying kimchi stuff. Is there a clean way to ignore this one in R1CS tests atm?

Reply:
Ah, the poseidon builtin hasn't been implemented for r1cs and this file was copied from the kimchi asm.